backend/cmd/jobs/env.go

```diff
@@ -35,6 +35,10 @@ type crowdinConfig struct {
 	ProjectIDs []int
 }
 
+type firebase struct {
+	ProjectID string
+}
```

Comment:
Can it implicitly use "current project id" if not set when running in GCP?

Reply:
Probably, which variable is that stored in?